creatures/alignment.cpp

```diff
@@ -332,6 +332,64 @@ void Player::alignAdjustAcThaco() {
 }
 
 
+bool Creature::mustRemainNeutral() {
+
+    switch(getClass()) {
+    case CreatureClass::DRUID:
+        return(true);
+        break;
+    case CreatureClass::CLERIC:
+        if (getDeity() == JAKAR || getDeity() == ARES)
+            return(true);
+        break;
+    default:
+        break;    
+    }
+    return(false);
+}
+
+bool Creature::mustRemainEvil() {
+
+    switch(getClass()) {
+    case CreatureClass::DEATHKNIGHT:
+    case CreatureClass::LICH:
+        return(true);
+        break;
+    case CreatureClass::CLERIC:
+        if (deity == ARAMON || deity == ARACHNUS)
+            return(true);
+        break;   
+    default:
+        break;
+    }
+    return(false);
+}
+
+bool Creature::mustRamainGood() {
+    switch(getClass()) {
+    case CreatureClass::PALADIN:
+    case CreatureClass::RANGER:
+        return(true);
+        break;
+    case CreatureClass::CLERIC:
+        if (deity == CERIS || deity == GRADIUS || deity == ENOCH ||
```

Comment:
Gradius and Ceris should go in neutral, no?

Reply:
They have been adjusted to need to stay good. No longer need to remain neutral. Adjusted other spots for them in the code too.